mod-orders-storage/examples/acquisitions_unit_membership_collection.sample

```diff
@@ -0,0 +1,21 @@
+{
+  "acquisitionsUnitMemberships":
+    [
+     {
+        "id": "39982f76-9738-4f49-b3a6-707d1e76853c",
+        "userId": "6e076ac5-371e-4462-af79-187c54fe70de",
+        "acquisitionsUnitId": "57c35c88-625d-4f0e-bc79-d22818d84d1c"
+     },
+     {
+        "id": "567a6d67-460f-43ab-af28-8c730c9aa4da",
+        "userId": "480dba68-ee84-4b9c-a374-7e824fc49227",
+        "acquisitionsUnitId": "f6d2cc9d-82ca-437c-a4e6-e5c30323df00"
+     },
+     {
+        "id": "874225ae-e3f5-4666-ae19-b35709ff0ee9",
+        "userId": "301883b4-a87b-4fc8-a4c1-e13813db06b0",
+        "acquisitionsUnitId": "aa0ec4e1-782f-45f6-a6f3-8e6b6c00599c"
+     },
```

Comment:
Could you remove the last comma, please.

Reply:
done